erpnext_documentation/www/docs/user/manual/en/customize-erpnext/desk-page.md

```diff
@@ -0,0 +1,52 @@
+<!-- add-breadcrumbs -->
+# Desk Page
+
+When you log in, you're presented with the Desk, it features a persistent sidebar sorted into categories. Each sidebar item links to a page called Desk Page.
+
+A Desk Page represents a module (for example CRM in ERPNext). A Desk Page includes the following:
+
+- A dashboard section for that particular module by default.
+- Shortcuts for commonly visited masters and pages
+- A masters section where all the reports and masters are grouped and listed
+
+<img alt="Print Format" class="screenshot" src="{{docs_base_url}}/assets/img/customize/standard-desk-page.png">
+
+## Standard Desk Page
+
+Every module in ERPNext has its own Standard Desk Page which is generated with appropriate shortcuts and links.
+
+<img alt="Print Format" class="screenshot" src="{{docs_base_url}}/assets/img/customize/desk-page-list.png">
+
+Any customization to be made to the Standard Desk Page can be done with Customize Desk Page option in the top right corner of the Desk Page.
+
+> Note: These customizations will be user specific and will only be visible to that user.
+
+<img alt="Print Format" class="screenshot" src="{{docs_base_url}}/assets/img/customize/customize-desk-page.png">
+
+## Custom Desk Page
+
+You can also create your own Desk Pages by simply creating new Desk Page document.
+
+<img alt="Print Format" class="screenshot" src="{{docs_base_url}}/assets/img/customize/new-desk-page.png">
+
+1. Go to the Desk Page list and click on New.
+
+1. **Name**: The text you enter here will be shown in the sidebar.
+
+1. **Module**: Select the module which the Desk Page will represents.
+
+1. ***Is Standard***: If checked, this Desk Page will be shown in the sidebar otherwise it will be treated as a customized version of other Desk Page.
```

Comment:
```suggestion
1. ***Is Standard***: If checked, this Desk Page will be shown in the sidebar otherwise it will be treated as a customized version of a Standard Desk Page.
```